integration_tests/dbt_project.yml

```diff
@@ -21,6 +21,16 @@ seeds:
   qualtrics_source_integration_tests:
     +column_types:
       _fivetran_synced: timestamp
+      directory_unsubscribe_date: timestamp
```

Comment:
was `creation_date` not set as `timestamp` because those seeds already had not null timestamps associated with them in the csvs?

Reply:
Exactly! Truthfully, I only added the fields that were causing problems in the integration tests. This step is only necessary to force the dbt seed to understand what the datatype of the column should be. I have a feeling `_fivetran_synced` was included previously because it was something we have done in other packages. It is not entirely necessary, but also doesn't cause any issues and should be fine to keep.